users/views.py

```diff
@@ -29,9 +40,29 @@ class UserRegistrationView(FormView):
 
 
     def form_valid(self: "UserRegistrationView", form: UserRegistrationForm) -> HttpResponse:
-        form.save()
-        email = form.cleaned_data.get("email")
-        messages.success(self.request, f"Account created for {email}!")
+        user = form.save(commit=False)
+        user.is_active = False
+        user.save()
+        uid = urlsafe_base64_encode(force_bytes(user.pk))
+        token = account_activation_token.make_token(user)
+        activation_link = f"http://{self.request.get_host()}/{uid}/{token}/"
```

Comment:
Hmm...
1. What is this activation_link for? I see it's used in send_email, but not in the email template. If you generated it then it should be used in both places. 
2. Also, I am missing here call to "reverse" or "reverse_lazy" to generate this url. 
3. The protocol should be https or http depending if we are on secure server...  An old topic: https://stackoverflow.com/a/21632907